packages/ai/src/ai/modules/mcp/apps/src/shared/theme.css

```
@@ -0,0 +1,229 @@
+ * Shared widget design-system. Neutrals are host-adaptive (currentColor
+ * mixes over the host's light/dark scheme); RocketRide identity lives in
+ * the accents: Horizon Blue #41b6e6 for interaction, the logo's flame
+ * #F93822 for upload/drag "energy" moments. Palette source:
+ * github.com/rocketride-org/branding (Palette.png).
+ */
+:root {
+	color-scheme: light dark;
+	--rr-accent: #41b6e6;
+	--rr-flame: #f93822;
+	--rr-flame-soft: #ff7a52;
+	--rr-line: color-mix(in srgb, currentColor 14%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)